### DISCUSSION TITLE
Tab Previews under the tab...

### DISCUSSION THREAD

**Discussion by MutantViking** [Floorp-Projects/Floorp#591] [Feature Request]
You have "Show tab previews in the Windows taskbar" but not under a tab like most Google browsers, for example Vivaldi browser in the gif below.
![vivaldi_YjvkGAd9JU](https://github.com/Floorp-Projects/Floorp/assets/151750707/fd533fd2-e19f-4afb-b44b-7b7fe6625910)

I tried https://addons.mozilla.org/ja/firefox/addon/tab-preview-on-hover/ and other stuff but nothing works and there's no feature like this on Floorp.

**Comment by Ryosuke-Asano** [comment]
Follow to install userChrome.css

![image](https://github.com/Floorp-Projects/Floorp/assets/73892113/14135d01-b06c-4739-bb7a-9ed0fe3489a3)

**Reply by MutantViking** [reply]
Yeah I did all that replaced the userChrome.css file but I think that extension is outdated which is why I'm making a feature request
![floorp_TXNYGvTsNC](https://github.com/Floorp-Projects/Floorp/assets/151750707/05072e00-876d-4506-afbd-c1bb07e03f23)

**Comment by kaos-55** [comment]
@MutantViking Check this out on the extension page: 
-Incompatible with built-in browser themes “System theme – auto” and “Firefox Alpenglow”
-Only the first ten tabs on the left will have tab preview

This is probably the problem. I just tested it in Firefox nightly and the extension works, if it works in nightly it shouldn't have a problem working with Floorp
![tab-preview](https://github.com/Floorp-Projects/Floorp/assets/86812371/31a1c870-9d50-4560-b607-28dcba3c1e56)

Still I support MutantViking if there was the possibility to implement the tab thumbnail in floorp it would be great, as the extension has several limitations like it only shows the thumbnail of the first 10 tabs and for example in Linux the extension does not work, i mentioned this problem to the extension developer but he could not give me a solution.

It is worth mentioning by the way that I am using the tab thumbnail in floorp through a script, so I think it should not be difficult to implement this feature. Just my 2 cents.
